src/main/java/com/google/cloud/tools/maven/RunMojo.java

```diff
@@ -80,7 +90,7 @@
    * Path to the data (datastore, blobstore, etc.) associated with the application. (default: None)
    */
   @Parameter(alias = "devserver.storagePath", property = "app.devserver.storagePath")
-  protected String storagePath;
+  protected File storagePath;
```

Comment:
It does, and I'm not aware of any effect. They provide a path string in both cases. https://maven.apache.org/guides/plugin/guide-java-plugin-development.html#Files_and_Directories